Common/src/CConfig.cpp

```diff
@@ -8864,12 +8893,34 @@ su2double CConfig::GetWall_HeatFlux(string val_marker) const {
   return Heat_Flux[iMarker_HeatFlux];
 }
 
+su2double CConfig::GetWall_HeatTransfer_Coefficient(string val_marker) const {
+  unsigned short iMarker_HeatTransfer = 0;
+
+  if (nMarker_HeatTransfer > 0) {
+  for (iMarker_HeatTransfer = 0; iMarker_HeatTransfer < nMarker_HeatTransfer; iMarker_HeatTransfer++)
+    if (Marker_HeatTransfer[iMarker_HeatTransfer] == val_marker) break;
+  }
+
+  return HeatTransfer_Coeff[iMarker_HeatTransfer];
+}
+
+su2double CConfig::GetWall_HeatTransfer_Temperature(string val_marker) const {
+  unsigned short iMarker_HeatTransfer = 0;
+
+  if (nMarker_HeatTransfer > 0) {
```

Comment:
The "ifs" are redundant since 0 is not smaller than 0.

Side note:
We have this search pattern in every method, but technically it results in out-of-bounds access if the string is not found.

Reply:
> The "ifs" are redundant since 0 is not smaller than 0.

true... someone wanted to make really sure that the loop-body is not executed 🛑  ... 7years ago (for e.g. heatflux).

I'll remove that for hf, isothermal and ht